cv_next/lib/definitions.ts

```diff
@@ -8,13 +8,44 @@ export default class Definitions {
 }
 
 export const heroHeader: PageHeader = {
-  header: `Landing pages made easy`,
-  subheader: `Easy to setup. Customizable. Quick. Responsive.`,
-  image: `/hero-img.webp`,
+  header: `Next`,
+  image: `/images/logo.png`,
+  subheader: `improve the magshimim next community's resumes.`,
+  explenetion: `we as a magshimim next community can help each other get jobs by improving our memeber's resumes.
+  this website is a place to uplode resumes, let other review them and make them better.
+  we will not tolerate any abuse, hateful commects, griffing, pentesting and etc... `
 }
 
 export const inactiveHeader: PageHeader = {
   header: `Access Denied`,
   subheader: `This page requieres that you get approved by the moderators`,
   image: `/access-denied.webp`,
 }
+
+
+export const routes: routes[] = [
```

Comment:
is there a way to make stuff like the signout and feed also appear on the popup? and to only show stuff like signout and profile to logged-in users?

Reply:
wdym?

Reply:
Is there a way to hide links from users based on if their logged in or logged out? i.e. users that are not signed in won't see sign out or profile and users that are signed in won't see login

Reply:
i think there is a way but lets do it after v1